File: HRO/TIRPRO01 Programmeren 1/o5.java
// opdracht 5; Priemgetallen; Zeef van Eratosthenes
public class o5
{
	public static void main(String[] args)
	{
		int i = 0; // index of the prime number
		boolean[] b = new boolean[2000]; // some random high enough number
		
		for (int p=2; (p<b.length && i<200); p++) // number to be checked
		{
			if (!b[p])
			{
				i++;
				System.out.println(i+"\t"+p);
				
				for (int j=p*2; j<b.length; j+=p)
					b[j] = true;
			}
		}
	}
}

File: HRO/TIRPRO01 Programmeren 1/Lua/o5.lua
-- opdracht 5; Priemgetallen; Zeef van Eratosthenes
local t = {}
local n = 2000 -- some random high enough number
local i = 0 -- index of the prime number

for p = 2, n do -- number to be checked
	if i >= 200 then
		break
	end
	
	if not t[p] then
		i = i + 1
		print(i, p)
		
		for j = p*2, n, p do
			t[j] = true
		end
	end
end
